pytorch_lightning/core/datamodule.py

```diff
@@ -184,7 +184,7 @@ def dims(self, d):
         rank_zero_deprecation("DataModule property `dims` was deprecated in v1.5 and will be removed in v1.7.")
         self._dims = d
 
-    def size(self, dim=None) -> Union[Tuple, int]:
+    def size(self, dim=None) -> Union[Tuple, List[Tuple]]:
```

Comment:
according to the doc above, dims is a tuple, but type is not annotated so it can be anything really. therefore type here is also arbitrary, right?

Reply:
Right.